codespell_lib/data/dictionary.txt

```diff
@@ -63596,8 +63596,8 @@ yuo->you
 yuor->your
 yur->your
 yymmetrical->symmetrical
-zar->czar
-zars->czars
+zar->czar, zarr
+zars->czars, zarrs
```

Comment:
`Zarr` is OK, but not sure about the plural `Zarrs`.

Don't forget the trailing comma.

Reply:
sorry, what trailing comma?

Reply:
Let's remove

```suggestion
```

Reply:
I guess not to leave it behind... ok let's do:

```suggestion
zars->czars
```